src/cr/cube/stripe.py

```diff
@@ -579,6 +579,7 @@ def count(self):
 
     @lazyproperty
     def mean(self):
+        """Return stripe of means."""
```

Comment:
This is a property, we don't need the "return" because we're trying to describe what it _is_ rather than what it _does_ (because it's a property :) )